app.py

```diff
@@ -493,6 +499,10 @@ def main():
     """
     options, args = parse_comand_line()
 
+    if options.remote_host is not None:
```

Comment:
Move the import here:

```python
from invesalius.net.remote_control import RemoteControl
```

Reply:
That's a good point. Done and tested that it works, this time both with and without the --remote-host command line argument.